src/main/java/io/strimzi/kafka/bridge/http/HttpSinkBridgeEndpoint.java

```diff
@@ -356,6 +356,13 @@ private void doAssign(RoutingContext routingContext, JsonObject bodyAsJson) {
         this.setAssignHandler(assignResult -> {
             if (assignResult.succeeded()) {
                 HttpUtils.sendResponse(routingContext, HttpResponseStatus.NO_CONTENT.code(), null, null);
+            } else {
+                HttpBridgeError error = new HttpBridgeError(
```

Comment:
why just returning CONFLICT? Couldn't it depend on the cause of error and why the assignment went wrong?
What kind of errors did you find?